src/js/controller/model.js

```diff
@@ -248,7 +248,8 @@ define([
 
         // Give the option for a provider to be forced
         this.chooseProvider = function(source) {
-            return _providers.choose(source).provider;
```

Comment:
@pajong this only happens if there is something wrong with playlist filtering. If you see this happen, you need to fix the provider-supports filtering so that the item is filtered out.